ColorStripe/Views/Devices/PeripheralModel.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  PeripheralModel.swift
+//  ColorStripe
```

Comment:
I prefer to remove class names and project name from headers, it's hard to support it in actual state. I'll add header template after PR merging.

Reply:
Done